### ISSUE TITLE
Eval

### ISSUE THREAD

**Issue by nod0n** [motion-synthesis-for-biomed-research/.github#40]
## Description

- edge testing, what are limitations of model?
- llm logits: is it often predicting same token?
- generate IMU data, we do the motion and measure it, compare to our IMU data
- Hypothesis testing
- Discussion: What could be done better next time (first implement working paper and then improve it)
